pubsub/rabbitpubsub/rabbit.go

```diff
@@ -118,8 +121,25 @@ func (o *URLOpener) OpenTopicURL(ctx context.Context, u *url.URL) (*pubsub.Topic
 
 // OpenSubscriptionURL opens a pubsub.Subscription based on u.
 func (o *URLOpener) OpenSubscriptionURL(ctx context.Context, u *url.URL) (*pubsub.Subscription, error) {
-	for param := range u.Query() {
-		return nil, fmt.Errorf("open subscription %v: invalid query parameter %q", u, param)
+	for param, value := range u.Query() {
+		switch param {
+		case "qos":
```

Comment:
Shouldn't this be called "prefetch_count"?

Reply:
That can be a good idea.

Reply:
done via 1c10a049fd44a97a27978c4bbf5ea9466886b8bf